integration-test/trace-recorder/src/main/resources/integrationTestData/kotlinx.collections.immutable/tests.contract.list.ImmutableListTest_empty.txt

```diff
@@ -439,32 +445,34 @@ ImmutableListTest.empty() at :0
               exceptionPredicate.invoke(actualFail): true at ComparisonDSL.kt:36
               assertTrue(true, "actual fail") at ComparisonDSL.kt:36
           CompareContext@1.propertyEquals("", CollectionBehaviorsKt$$Lambda@1) at CollectionBehaviors.kt:23
-            getter.invoke(this.expected): -1 at ComparisonDSL.kt:21
+            getter.invoke(expected): -1 at ComparisonDSL.kt:21
               getOrNull(propertyEquals, 0): null at CollectionBehaviors.kt:23
               indexOf(propertyEquals, null): -1 at CollectionBehaviors.kt:23
-            getter.invoke(this.actual): -1 at ComparisonDSL.kt:21
+            getter.invoke(actual): -1 at ComparisonDSL.kt:21
               getOrNull(propertyEquals, 0): null at CollectionBehaviors.kt:23
               indexOf(propertyEquals, null): -1 at CollectionBehaviors.kt:23
             assertEquals(-1, -1, message ➜ "") at ComparisonDSL.kt:21
           CompareContext@1.propertyEquals("", CollectionBehaviorsKt$$Lambda@1) at CollectionBehaviors.kt:24
-            getter.invoke(this.expected): -1 at ComparisonDSL.kt:21
+            getter.invoke(expected): -1 at ComparisonDSL.kt:21
               getOrNull(propertyEquals, 0): null at CollectionBehaviors.kt:24
               lastIndexOf(propertyEquals, null): -1 at CollectionBehaviors.kt:24
-            getter.invoke(this.actual): -1 at ComparisonDSL.kt:21
+            getter.invoke(actual): -1 at ComparisonDSL.kt:21
               getOrNull(propertyEquals, 0): null at CollectionBehaviors.kt:24
               lastIndexOf(propertyEquals, null): -1 at CollectionBehaviors.kt:24
             assertEquals(-1, -1, message ➜ "") at ComparisonDSL.kt:21
           listBehavior.propertyFails(CollectionBehaviorsKt$$Lambda@1) at CollectionBehaviors.kt:26
-            CompareContext@1.assertFailEquals(CompareContext$$Lambda@1, CompareContext$$Lambda@1, null) at ComparisonDSL.kt:23
+            assertFailEquals(CompareContext$$Lambda@1, CompareContext$$Lambda@1, null) at ComparisonDSL.kt:23
```

Comment:
Double `$` might be a bug here

Reply:
I checked, it is not bug, it is real class name.